### ISSUE TITLE
DWG Review:  Rename 'cutout tool

### ISSUE THREAD

**Issue by sliptonic** [shaise/FreeCAD_SheetMetal#327]
This is suggested to be named “Cutout” it is better descriptive than “sketch on sheet metal” which has 
poor discoverability.

No Task panel is needed for this function currently.

![cutouttool](https://github.com/shaise/FreeCAD_SheetMetal/assets/538057/2568c0a3-cd3a-49c1-9a22-c1cba2db1128)

**Comment by shaise** [comment]
Definitely should be renamed. Also to minimize workflow confusion, we should not show the pocket tool of the part design.

**Comment by maxwxyz** [comment]
Is there a way to specify cutting depth currently? I can remember I had the problem once, that it did not cut through the entire sheet thickness. Maybe a user also does not want cut through all.

**Comment by shaise** [comment]
I think that in sheetmetal, the cut should always be through all. Otherwise it will not be a part that can be laser cut from a flat sheet. If, for any reason the user wants to make such operation, he can always do it with the part design pocket feature.

**Comment by pierreporte** [comment]
@shaise The change has been made so I think the issue can be closed.

**Comment by shaise** [comment]
Right now we have both tools. I plan to phase out "sketch on sheetmetal", so, I believe we can close it

**Comment by pierreporte** [comment]
What do you mean by phasing out the feature? Will there be one to make cutouts in unfolded state instead?

**Comment by shaise** [comment]
There are 2 tools now: 
1. Sketch on sheetmetal: this is essentially a duplication of the PD pocket tool. 
2. Extruded cutout. The new tool.

Both tools are quite similar - however the first tool, when cutting a diagonal flange will create diagonal cuts - which creates a non 'legal' sheetmetal part, and the new one ensures the the cuts are always 90 degrees to the sheet plane.
I want to remove the first tool. It will still stay in the code so old files can be loaded, but will not show on the toolbar

**Comment by pierreporte** [comment]
But it will remove the ability to wrap the cutout around the bent part, as if the flat sheet was cut and then bent. The ability to do that shouldn't disappear IMO. It could be a good idea to integrate it into the new tool, with an option to cut the bent part or the flat sheet, while allowing sketches everywhere and not only a face for both modes. An alternative way would be to make un unfolded part a status that can be on and off, to allow user to manually chose (as in Catia).

**Comment by shaise** [comment]
Oh, I just found out what yo mean... sorry about that, I was under the impression that the hole is just projected. the sketch actually curves with the bends. 
You are right. Its two different tools. They will both stay. 
So back to square one. do we need to change the name?

**Comment by sheetmetalman** [comment]
Hi, guys!

I think could has an option inside an main tool. Or merge the two tools into a dropdown button.

ExtrudedCutout can make the same on a flat face as the SketchOnSheet. The main difference is when the surface isn't flat. Because of this I think the SketchOnSheet can be renamed as "wrap cut" or something like this. And this is very nice, as far as I remember not even SolidWorks has tool like this for sheet metal.

There's just an issue where if the sketch has unconnected wires (still closed of course), the tool only cuts the one that touches the flat face of the sheet metal.

https://github.com/user-attachments/assets/6bbd0caa-7c68-4b43-82a4-c759a96ea9ac


https://github.com/user-attachments/assets/6e691f87-af32-41f7-a079-2be63ece01e1

**Comment by shaise** [comment]
@sheetmetalman , Yes, I think we only need to rename it to Wrap Cutout and perhaps change the Icon

**Comment by pierreporte** [comment]
Renaming to wrap cutout is a good idea. It’s more descriptive.

Merging the two tools is also a good idea, I even suggested it, because it will allow for sketches that are not on a face (the user would still need to select one though). However it can be done later.

@sheetmetalman You should open an issue for these bugs.

**Comment by FC-FBXL5** [comment]
I like wrap cutout, too, but I would like to keep the tools separate as they are two completely different tools.

**Comment by shaise** [comment]
> I like wrap cutout, too, but I would like to keep the tools separate as they are two completely different tools.

I agree completely

**Comment by sheetmetalman** [comment]
> I like wrap cutout, too, but I would like to keep the tools separate as they are two completely different tools.

I agree too.
